### PR TITLE
st-theme 1.2.3 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
st-theme 1.2.3 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5415]
- dev_url:        https://github.com/gabrieltempass/streamlit-theme
- conda_forge:    None
- pypi:           https://pypi.org/project/st-theme/1.2.3
- pypi inspector: https://inspector.pypi.io/project/st-theme/1.2.3/
- related pr: https://github.com/AnacondaRecipes/streamlit-feedstock/pull/11

### Explanation of changes:

- initial recipe creation

[PKG-5415]: https://anaconda.atlassian.net/browse/PKG-5415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Arishamays1 — APPROVED** [review]

**Comment by ifitchet** [comment]
The `README.md` seems a bit short?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/st-theme-{{ version }}.tar.gz
+  sha256: ca97aece1a48ded6e83fd742c27cb0851e1bce2100ab4b6c37c7b6e003b65b42
+
+build:
+  entry_points:
+    - st-theme = streamlit_theme:print_version
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [s390x or py<=38]
```
Comment (ifitchet) [inline recipe/meta.yaml]: `setup.py` says `python_requires=">=3.8"`?:

```suggestion
  skip: true  # [s390x or py<38]
```

**Comment by Jrice1317** [comment]
> The `README.md` seems a bit short?

Here's a template you can use from the [template repository](https://github.com/AnacondaRecipes/template-repository) I created just a couple weeks ago. Hopefully this helps 😀 

About <pkg_name>-feedstock
=======================

Feedstock license: [BSD-3-Clause](LICENSE)

Home: <home_url>

Package license: <pkg_license>

Summary: <pkg_summary>


Current release info
====================

| Name | Downloads | Version | Platforms |
| --- | --- | --- | --- |
| [Conda Recipe](https://anaconda.org/anaconda/<pkg_name>) | [Conda Downloads](https://anaconda.org/anaconda/<pkg_name>) | [Conda Version](https://anaconda.org/anaconda/<pkg_name>) | [Conda Platforms](https://anaconda.org/anaconda/<pkg_name>) |

Installing <pkg_name>
==================

Installing `<pkg_name>` from the main channel can be achieved by:

```
conda install <pkg_name>
```

It is possible to list all of the versions of `<pkg_name>` available on your platform with `conda`:

```
conda search <pkg_name>
```

**Review by Jrice1317 — APPROVED** [review]
LGTM outside of addressing Ian's comment.

File: abs.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+channels:
+- psteyer-tests
+upload_without_merge: True
```
Comment (ifitchet) [inline abs.yaml]: This _is_ heading to Snowflake, right?

Comment (psteyer) [reply abs.yaml]: Yup! I've removed the abs information and added back in snowflake. This was just for me to test to make sure everything builds correctly.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  summary: A component that returns the active theme of the Streamlit app.
+  description: |
+    The function immediately returns the active theme, 
+    when it is called. If the user manually changes the theme, 
+    after the web app is already running, it updates the returned value.
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  dev_url: https://github.com/gabrieltempass/streamlit-theme/
+  doc_url: https://github.com/gabrieltempass/streamlit-theme/blob/main/README.md
```
Comment (ifitchet) [inline recipe/meta.yaml:49]: `setup.py` is saying use the GH repo: https://github.com/gabrieltempass/streamlit-theme/blob/1.2.3/setup.py which would present you with the README as a side-effect.  🤷

**Review by ifitchet — APPROVED** [review]
A nit about the `doc_url`

**Pull request merged**